Configurations.md

```diff
@@ -2008,6 +2008,56 @@ use dolor;
 use sit;
 ```
 
+## `group_imports`
+
+Discard existing import groups, and create three groups for:
+1. `std`, `core` and `alloc`,
+2. external crates,
+3. `self`, `super` and `crate` imports.
+
+Within each group, imports are sorted as with `reorder_imports`.
+
+This has no effect is `reorder_imports` is `false`.
+
+- **Default value**: `None`
+- **Possible values**: `None`, `StdExternalCrate`
+- **Stable**: No
+
+#### `None` (default):
```

Comment:
Let's keep the description of the option generic about what the option does/what it controls as opposed to describing the behavior of a particular supported variant (e.g. `Controls the strategy for how imports are grouped together...`)

And given the denotations of `None` within Rust, I'd suggest we use `Preserve` instead, which will also be consistent with a few other rustfmt config option variants